src/app/static/src/tests/components/runReport/reportList.test.ts

```diff
@@ -29,6 +30,12 @@ describe("reportList", () => {
         done();
     });
 
+    it("renders correctly and fires event on mount", () => {
```

Comment:
We should add an extra test here to ensure that an event is not fired if the report name is not valid